src/dtx/dtx_srv.c

```diff
@@ -309,6 +309,24 @@ dtx_init(void)
 		dtx_yield_cycle = DTX_YIELD_DEF;
 	}
 
+	str = getenv("DTX_AGG_THRESHOLD");
+	if (str != NULL) {
+		dtx_agg_thd_cnt_up = atoi(str);
+		if (dtx_agg_thd_cnt_up < DTX_AGG_THD_CNT_MIN ||
+		    dtx_agg_thd_cnt_up > DTX_AGG_THD_CNT_MAX) {
+			D_WARN("Invalid DTX aggregation threshold %d, "
+			       "the valid range is [%d, %d], use the "
+			       "default value %d\n",
+			       dtx_agg_thd_cnt_up, DTX_AGG_THD_CNT_MIN,
+			       DTX_AGG_THD_CNT_MAX, DTX_AGG_THD_CNT_DEF);
+			dtx_agg_thd_cnt_up = DTX_AGG_THD_CNT_DEF;
+		}
+	} else {
+		dtx_agg_thd_cnt_up = DTX_AGG_THD_CNT_DEF;
+	}
+
+	dtx_agg_thd_cnt_lo = dtx_agg_thd_cnt_up * 6 / 7;
```

Comment:
For my own understanding, why `6/7`?

Reply:
When DTX entries count exceeds the dtx_agg_thd_cnt_up, assume it is 7M, then DTX aggregation will be triggered that will remove some old DTX entries until the DTX entries count less than dtx_agg_thd_cnt_lo which is 7M * 6 / 7 = 6M. The configuration will avoid DTX aggregation ULT to be triggered too frequently.

Reply:
That makes sense. I guess another question is why 6/7 and not 5/7, or 5/6, etc.? Will this ratio need to be tested more in the future? (not now)

Reply:
Honestly, I do not know which one is better, 6/7 or 5/6 or others, just some intuitively choose. If you have chance in future, please help to give more measurement for that. Thanks!

Reply:
Sure. Once we get through the basics of this PR and some others related to DTX, we can look more closely at things like this.